dhis-2/dhis-services/dhis-service-schema/pom.xml

```diff
@@ -64,6 +106,11 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.hisp.dhis</groupId>
+      <artifactId>dhis-support-test</artifactId>
```

Comment:
Scope test?

Reply:
set in parent

https://github.com/dhis2/dhis2-core/blob/92973bfa27592e13215ad38a10822b7d0509a0b2/dhis-2/pom.xml#L823-L828

but I'll make it explicit 👌🏻